README.md

```diff
@@ -18,6 +18,161 @@ The connector needs the following permissions to the specified bucket:
 
 In case of ``Access Denied`` error see https://aws.amazon.com/premiumsupport/knowledge-center/s3-troubleshoot-403/
 
+### File name format
+
+The connector uses the following format for output files (blobs):
+`<prefix><filename>`.
+
+`<prefix>`is the optional prefix that can be used, for example, for
+subdirectories in the bucket. 
+`<filename>` is the file name. The connector has the configurable
+template for file names. 
+
+    Configuration property `file.name.template`. If not set, default template is used: `{{topic}}-{{partition}}-{{start_offset}}`
+
+It supports placeholders with variable names:
+`{{ variable_name }}`. Currently supported variables are:
+- `topic` - the Kafka topic;
+- `partition` - the Kafka partition;
+- `start_offset:padding=true|false` - the Kafka offset of the first record in the file, if `padding` sets to `true` will set leading zeroes for offset, default is `false`;
+- `timestamp:unit=YYYY|MM|dd|HH` - the timestamp of when the Kafka record has been processed by the connector.
+   - `unit` parameter values:
+     - `YYYY` - year, e.g. `2020`
```

Comment:
Oh, my. Here's a problem: `YYYY` and `yyyy` are different things in the date formatting, and people usually need `yyyy`. It seems we have this problem in the GCS connector as well. We can merge this as is, but we need to fix this (and the code, of course) before the next release.

Reply:
Solved everything except this one